### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by francknouama** [gangstead/atom-scala-format#5]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.

```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:54:11)
Object.activate (/Users/francknouama/.atom/packages/scala-format/lib/scala-format.js:20:21)
```

**Comment by gangstead** [comment]
Dupe of https://github.com/gangstead/atom-scala-format/issues/3 which I'm working on.

**Comment by gangstead** [comment]
Fixed in 0.1.3 with Atom 0.181 or above.

**Comment by francknouama** [comment]
Thanks @gangstead for the fix
